sup3r/postprocessing/collection.py

```diff
@@ -213,7 +213,8 @@ def get_data(self, file_path, feature, time_index, meta, scale_factor,
                 logger.error(e)
                 raise KeyError(e)
 
-            f_data = f[feature][...]
+            mask = f_meta['gid'].isin(meta['gid'])
+            f_data = f[feature][:, mask]
```

Comment:
Is this the strategy for doing the sparse collection? We should talk about this... The sparse final meta will likely have a gid column that is enumerated (reset) and is not sparse.

Reply:
The initial collection on the entire file list does a masking to sparsify the gids though